### DISCUSSION TITLE
Previously authored commits still show up in the timeline

### DISCUSSION THREAD

**Discussion by Apacelus** [community/community#51858] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

For a while one of my github actions was creating commits and setting me as author. I have now disabled that and have edited all commits where I was the author and the bot was the committer and made the bot the sole committer and author. I have done this about a month ago, so any cache should've been updated by now. 
The issue is, those commits still show up on my personal page and count towards my total commits (making them an unrealistic 5.8k number).
Here is an example:
![image](https://user-images.githubusercontent.com/46828271/229566960-23b8d207-2a78-4132-b0f8-987a56171ffe.png)
When I click on the rpm-repo link, I am redirected to this [page](https://github.com/eupnea-linux/rpm-repo/commits?author=Apacelus&since=2022-12-04&until=2022-12-05) which contains ~40 commits instead of 126.

**Comment by RamtinKosari** [comment]
Hi Dear @Apacelus 
there are a few steps you can take to try and fix this issue:

Clear local git cache: Sometimes, Git may cache information about your commits locally. To ensure that you're seeing the updated information, clear the local git cache on your machine. You can do this by running the following command in your terminal:
```shell
git rm --cached -r .
```
This will clear the cache and force Git to fetch the latest commit information from the remote repository.

Force push to update the remote repository: After clearing the local git cache, you can force push your local repository to update the remote repository with the corrected commit information. Before doing this, make sure you have a backup of your repository or relevant branches, just in case. To force push, use the following command:

```shell
git push -f origin <branch-name>
```
Replace <branch-name> with the name of the branch you want to update.

Verify the updated commits on GitHub: After force pushing, visit your repository on GitHub and check if the commits have been updated with the correct bot as the sole committer and author. It may take a few minutes for the changes to be reflected on the website.

If the above steps don't resolve the issue, it's possible that there might be another configuration or caching issue specific to GitHub. In that case, you should reach out to GitHub support for further assistance. They will be able to investigate the issue and provide guidance on how to fix it.

**Reply by Apacelus** [reply, marked as answer]
The issue was with GitHub and contacting support resolved it

**Reply by RamtinKosari** [reply]
Nice
